can-view-callbacks.js

```diff
@@ -117,7 +119,11 @@ var callbacks = {
 
 		//!steal-remove-start
 		if (!tagCallback) {
-			dev.warn('can-view-callbacks: No custom element found for ' + tagName);
+			var GLOBAL = getGlobal();
+			var ceConstructor = GLOBAL.document.createElement(tagName).constructor;
+			if (ceConstructor === GLOBAL.HTMLElement && ceConstructor === GLOBAL.HTMLUnknownElement) {
```

Comment:
I don't see how this statement would ever be true. Is there a test showing that the warning is given if a custom element is not defined?

Reply:
Oops! That was meant to be an `||`. I've added a test for this. Good catch!